src/services/emptyDocuments.ts

```diff
@@ -96,7 +96,6 @@ const emptyDocuments: Document[] = [
     eventStatus: "CLOSED",
     preResources: [],
     postResources: [],
-    eventSurvey: [],
```

Comment:
Maybe want to include this in the emptyDocument, even if its just as "Enter survey link here..." or just ""